app/src/components/bucket-list.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { connect } from 'redux-bundler-react'
+
+
+function BucketList (props) {
+  let buckets = ['ci-kubernetes-e2e-gci-gce/1116453304528801792',
+                 'ci-kubernetes-e2e-gce-cos-k8sstable3-default/1116369525902675974',
+                 'ci-kubernetes-e2e-gce-cos-k8sstable2-default/1116461866978119689',
+                 'ci-kubernetes-e2e-gce-cos-k8sstable1-default/1116466143159128064',
+                 'ci-kubernetes-e2e-gce-cos-k8sbeta-default/1116430159767932928'
+                ]
```

Comment:
Noting that we will need to make this dynamic at some point... maybe pulling from a file in gs://apisnoop/